ArchiSteamFarm/Actions.cs

```diff
@@ -244,7 +244,7 @@ public sealed class Actions : IAsyncDisposable {
 		}
 
 		[PublicAPI]
-		public async Task<(bool Success, string Message)> SendInventory(IReadOnlyCollection<Steam.Asset> items, ulong targetSteamID = 0, string? tradeToken = null) {
+		public async Task<(bool Success, string Message)> SendInventory(IReadOnlyCollection<Steam.Asset> items, ulong targetSteamID = 0, string? tradeToken = null, ushort itemsPerTrade = Trading.MaxItemsPerTrade) {
```

Comment:
We should have `< 2` check here as well, it's not a problem that it's checked twice, and the earlier we throw, the better.